app/src/main/res/layout/item_view.xml

```diff
@@ -2,11 +2,12 @@
 <androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:app="http://schemas.android.com/apk/res-auto"
     android:layout_width="match_parent"
-    android:layout_height="@dimen/item_height">
+    android:layout_height="wrap_content"
+    android:minHeight="@dimen/item_height">
 
     <TextView
         android:id="@+id/title_text"
-        android:layout_width="wrap_content"
+        android:layout_width="match_parent"
```

Comment:
If you are using `ConstraintLayout` as a parent ViewGroup, you shouldn't use `match_parent`, it has its own version called match constraint. You need use it instead.